external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchSourceSuite.scala

```diff
@@ -2429,8 +2429,9 @@ class KafkaSourceStressSuite extends KafkaSourceTest {
       (d, running) => {
         Random.nextInt(5) match {
           case 0 => // Add a new topic
-            topics = topics ++ Seq(newStressTopic)
-            AddKafkaData(topics.toSet, d: _*)(message = s"Add topic $newStressTopic",
```

Comment:
So, previously, is `newStressTopic` function invoked twice?

Reply:
Looks like. Nice finding @viirya !

Reply:
Yea, that is why we see the error message looks like:


```
AddKafkaData(topics = Set(stress4, stress6, stress2, stress1, stress5, stress3), data = Range 15 until 20, message = Add topic stress7)
```